client/engine/chainservice/eth_chainservice.go

```diff
@@ -59,6 +60,21 @@ const MAX_QUERY_BLOCK_RANGE = 2000
 // See https://github.com/ethereum/go-ethereum/blob/e14164d516600e9ac66f9060892e078f5c076229/eth/filters/filter_system.go#L43
 const RESUB_INTERVAL = 2*time.Minute + 30*time.Second
 
+// NewEthChainService2 is a convenient wrapper around NewEthChainService, which provides a simpler API
+func NewEthChainService2(chainUrl, chainPk string, naAddress, caAddress, vpaAddress common.Address, logDestination io.Writer) (*EthChainService, error) {
```

Comment:
Open to suggestions for a better name. 

Initially I had wanted to replace (not wrap) `NewEthChainService`... but this causes problems:
* the simulate back end chain service 
* the FEVM tests
want access to the inner constructor.

Reply:
I think the simulated and FEVM tests are local to the `chainservice` package, so could we change the old constructor to a private one?

Reply:
Oh that would be nice. Looks like maybe testground uses it, too -- but I haven't dug in to see if that would be easy to fix.

Reply:
These are some changes that should get things working again with testground https://github.com/statechannels/go-nitro-testground/pull/166

Reply:
> I think the simulated and FEVM tests are local to the `chainservice` package, so could we change the old constructor to a private one?

[3a3f91e](https://github.com/statechannels/go-nitro/pull/1279/commits/3a3f91e6384fe7c0e3b4213373a856d0bf13ae1d)